doc/user/config.toml

```diff
@@ -3,6 +3,7 @@ title = "Materialize Documentation"
 pygmentsCodeFences = true
 pygmentsUseClasses = true
 
+disableKinds = ['taxonomy']
```

Comment:
Not sure if we need to include `term` in the array. So, for now, just specified taxonomy.